### ISSUE TITLE
skins

### ISSUE THREAD

**Issue by Pepelnica1488** [0x000015/SpaceEngineers-DLC-Bypass#7]
can i change my ship skin with that files on multiplayer servers?

**Comment by 0x000015** [comment]
Skins can't be unlocked using simple XML edits sadly. So no.
Sorry.

**Comment by Pepelnica1488** [comment]
can you show how to unlock that shit? ;d

**Comment by 0x000015** [comment]
That'd be beyond me, sorry.

**Comment by Pepelnica1488** [comment]
fuk=(()( kins are faggots

**Comment by Kanken6174** [comment]
i somehow wonder... Buying the dlcs download new meshes files for the skins that are 500mb for each, i wonder if there is an activator in there that says if you can use the paints or not... 

(i am talking about the blocks paint here and not the actual engineer suits you get from the dlc, those are part of your inventory and as such its impossible to get them with client-sided edits, because its handeled via steam)

i am still thinking about testing some stuff with these mesh files, maybe i can somehow get them to work on an account that hasnt bought the dlc. What do you think 0x000015?

**Comment by 0x000015** [comment]
Well since block skins are in your inventory as well I don't believe you can fake it so there may be a check for that.

**Comment by Kanken6174** [comment]
yes you're right i didn't notice at first. That means that unless we use some kind of duplication/forced exchange of the key item, we cant get the paints... too bad

**Comment by shadenquest** [comment]
Need an update on the unlock instructions,  old folders don't exist now. I only play offline,  no internet way out here.  And my phone can barely download anything after hours of waiting. Just need what i paid for unlocked online,  not this stupid BS where they steal your stuff if you don't have great internet on all the time.
